packages/rs-dpp/src/data_contract/state_transition/data_contract_update_transition/validation/state/validate_data_contract_update_transition_state.rs

```diff
@@ -65,7 +67,8 @@ pub async fn validate_data_contract_update_transition_state(
         .map_err(Into::into)?;
 
     if state_transition.execution_context.is_dry_run() {
```

Comment:
All other validation functions now return actions? Doesn't the validation function should only validate?